src/Components/Common/components/ButtonV2.tsx

```diff
@@ -105,7 +105,7 @@ const ButtonV2 = ({
 }: ButtonProps) => {
   const className = classNames(
     props.className,
-    "font-medium h-min inline-flex whitespace-pre items-center gap-2 transition-all duration-200 ease-in-out cursor-pointer disabled:cursor-not-allowed disabled:bg-gray-200 disabled:text-gray-500 outline-offset-1",
+    "font-medium h-min w-full md:w-fit inline-flex whitespace-pre items-center gap-2 transition-all duration-200 ease-in-out cursor-pointer disabled:cursor-not-allowed disabled:bg-gray-200 disabled:text-gray-500 outline-offset-1",
```

Comment:
Adding size classes here will affect all buttons used in the platform. Size classes were not put explicitly to allow flexibility based on where it's used.

Discard this changes and pass `w-full md:w-fit` in classNames instead from where it is required.

Reply:
I am really sorry its my first time contributing to open source. Can u please tell me how to put labels, there were only 9 labels available, do I have to create a new label? And you were right I ll make changes and create a new PR. @rithviknishad

Reply:
Not sure what you meant by "labels". Labels can be managed by members with triage permission only. We'll take care of that 😄 

If you meant closing keywords, have a look at this and other PRs and how it's used. https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword

Reply:
I am a bit confused but I have made some changes please review @rithviknishad